### ISSUE TITLE
US-Soviet Joint Projects

### ISSUE THREAD

**Issue by settintotrieste** [settintotrieste/Victoria-2-Cold-War-Enhancement-Mod-CWE#19]
Main feature for 1.06

**Comment by ARR8** [comment]
Some possibilities:
- Joining the ISS project could give relations with other members, easy change
- After a certain point, and with low tensions, the bloc leaders could let members of their aid systems (Marshall plan, COMECON, also possibly international) apply to aid from both powers - joint economic development, investment in the third world. 
- If both have similar attitudes to climate change, joint environmental projects or experiments, especially to do with empty land in Siberia and Western America
- Events to do with Olympics if both are involved and neither caught doping, visits from teams, games between the powers
- Events between the capitalist bloc leader and any market communist state to foster relations and trade (incl. China)

**Comment by settintotrieste** [comment]
Those are excellent ideas! I will try to get as many of them in as possible.

**Comment by ARR8** [comment]
Great! Just submitted a quick patch for the first one, and I'll look out for other suggestions.

**Comment by settintotrieste** [comment]
Thanks!

**Comment by ARR8** [comment]
Other ideas:
- If the treaties after SDI development are completed, or at least just SALT, the two could work on global disarmament. I doubt any power would agree to remove nuclear weapons completely, but I think some of the arms race items could begin to be rolled back worldwide.
- If the USSR begins Glasnost' reforms, there could possibly begin a student exchange between the countries, something that could encourage liberalism and democracy in the USSR and sympathy for socialism (once #50 is fixed and it becomes less powerful) in the US.
- Beyond the interpol, there could be counterterrorism efforts: the USSR historically put in a lot of effort suppressing Islam within the country. Possibly one of the ways to end the war on terror could be the adoption of the Soviet tactic worldwide by both blocs of pushing a moderate, decentralized Islam to weaken radical movements. However, there were certainly large differences between the powers on their ideas about religion, so this should probably be very difficult.

**Comment by settintotrieste** [comment]
Nuclear disarmament that reduced the size of the arsenal which was what the superpower this would be difficult to model into the game because currently the nuclear system is based off delivery capabilities instead of arsenal size so I probably wouldn't want to add that in. Currently nuclear acquisition is limited by the nonproliferation treaty which most nations will sign.

This is a good idea. I will put it under a "US-Soviet cultural outreach" decision.

I think to make a feature to suppress radical Islam in the game to stop the war on terror might make it look like all the terrorists are Islamic fanatics which will misrepresent the real life situation where there were many different types of terrorists, the secular PLO, IRA, basque terrorists, Maoists in India etc. Perhaps a better way would be to have a generic Superpower joint suppression of "radical extremism" decision that would take about 25 years to end the war on terror.

**Comment by ARR8** [comment]
For the third one: this is a good point, but I think it would be too simple to discount the effect of radical Islam on terror. Maybe far-right Islamic nations can choose to sponsor terror for a minor increase in occurrences and benefits to them (pacify people, prestige?) but that can be stopped if liberal democracy or communism is enforced.
  For example, all of the terrorist attacks claimed by ISIS, as the major sponsor of terrorism, can't be modeled in-game. Plus, many Muslim nations were formerly parts of the USSR, so all of those would add difficulty as more nations to be managed by a superpower.

Again, I bring it up because it was the historical Soviet policy, although this should honestly be a separate issue. I agree on all your other points.

**Comment by settintotrieste** [comment]
I agree with your point and as a middle ground solution we can have the Islamic nations with populist, traditionalist or nationalist ruling parties start choosing whether to sponsor terror when ISIS or 9/11 occurs with the trade offs and benefits along the lines of your proposal with the enforcement of liberal democracy or communism needing to stop the promotion. The superpowers will then have the option to force the cessation of such activity via sphering or puppeting the sponsors. This would simulate the role of bipolarity in keeping a lid over local problems.

**Comment by ARR8** [comment]
Thanks, that's exactly what I meant. I'll make a separate issue for it.
